Libraries/Vibration/Vibration.js

```diff
@@ -27,11 +27,31 @@ var Platform = require('Platform');
  */
 
 var Vibration = {
-  vibrate: function(duration: number = 400) {
+  vibrate: function(pattern: number | Array<number> = 400, repeat: number = -1) {
```

Comment:
@dmmiller Oh wait, it's the index of the pattern, not repeat n times. I think we should just change it to `true | false` and pass `0 | -1` to Android accordingly.